app/models/group.rb

```diff
@@ -31,7 +31,7 @@ class Group
     #   'realm2' => ['perm3', 'perm4', ...] }
     field :minecraft_permissions, type: Hash, default: {}.freeze
 
-    Flair = Struct.new(:symbol, :color)
+    Flair = Struct.new(:symbol, :color, :priority)
```

Comment:
While you're changing this, you might want to allow multiple colors to be defined. We wanted to try out bold flairs, but this was not possible given the one color code format.